api/src/templates/new_member.body.html

```diff
@@ -1,167 +1,194 @@
+<h3>Trivselregler</h3>
 
-<h3>Jag lämnade mitt projekt på ett bord över natten och sen var det borta!</h3>
-<p>Du måste städa efter dig. Vi har en städare som vi betalar en ansenlig summa för att våttorka golv och bord varje vecka. Men han rör inte bord med saker på, och städar inte golv där det står saker. Det är jätteviktigt att du städar undan efter dig, så att vi kan ha en gemensam, ren lokal. Har du råkat lämna saker på ett bord så kan det ha slängts eller flyttats till en Lost&Found-låda. Medlemmar hjälps åt att tömma soporna. Fråga en medlem om du inte vet hur man gör, eller kolla in <a href="https://wiki.makerspace.se/St%C3%A4drutiner">städrutinerna</a>.</p>
+<p style="text-align:center;"><i>“Visa hänsyn. Ta ansvar.”</i></p>
 
-<h3>Jag är hungrig.</h3>
-<p>Stockholm Makerspace erbjuder ett pentry med en kaffekokare och mikrovågsugn. Du kan också köpa färdigmat på The Coffice över gatan, eller på 7-Eleven eller T-Snabben nere vid Östra Station.</p>
+<p>Visa hänsyn mot andra medlemmar:</p>
+<ul>
+    <li>Städa alltid upp efter dig. Glöm inte att fler medlemmar kommer senare för att använda samma ytor och maskiner. Den medlemmen kan vara du.</li>
+    <li>Se till att använda utrustning och lokaler på ett korrekt sätt. Om utrustning används på felaktivt vis kan du skada utrustningen och göra den omöjlig att använda för fler medlemmar. Den medlemmen kan vara du.</li>
+    <li>Informera om något är trasigt, försvunnet eller om det behöver fyllas på med något. Saker går sönder ibland, det är sånt som händer. Om du inte kan laga det själv så skriv på Slack. Om du inte gör det kommer utrustningen förbli trasig när fler medlemmar vill använda den. Den medlemmen kan vara du.</li>
+    <li>Ta inte utrustning med dig. Om du tar hem saker är den inte tillgänglig för fler medlemmar. Den medlemmen kan vara du.</li>
+</ul>
 
-<h3>Hur funkar styrelsen?</h3>
-<p>Styrelsen arbetar ideellt och möts ungefär 1 gång i månaden. Då går man igenom ekonomi, kommande medlemsintroduktions-datum, eventuella problem och tar beslut kring större inköp. Om du vill kontakta styrelsen så emaila <a href="mailto:info@makerspace.se">info@makerspace.se</a>.</p>
+<p>Ta ansvar för...</p>
+<ul>
+    <li>...dina projekt och ditt material. Om du lämnar projekt eller material utan att märka upp det så kommer det slängas.</li>
+    <li>...dina gäster. Du får ta in gäster för att visa lokalerna men de får inte arbeta på dina projekt. Barn under 16 år får vara med och kan arbeta på projekt under ditt ansvar.</li>
+    <li>...din säkerhet. Du ansvarar för din fysiska säkerhet i föreningens lokaler.</li>
+    <li>...din användning av maskiner och utrustning. Du har ansvar för att du använder dessa på ett säkert sätt. Läs alltid de tillhörande skyltarna och informationen på wikin innan du sätter igång.</li>
+    <li>...att följa vår nolltolerans mot brott. Avsteg leder till påföljder.</li>
```

Comment:
Känns lite implicit?

Reply:
Yes... and still, things happen. I would leave it in to be extra clear about it, but I am open to hearing any reason to take it out.

Reply:
I feel like it's redundant given that it's already illegal. People doing illegal stuff aren't going to be discouraged by a note in an email. And a shorter email means people are more likely to read and absorb the important bits.